extensions/positron-python/src/client/positron/session.ts

```diff
@@ -105,7 +105,9 @@ export class PythonRuntimeSession implements positron.LanguageRuntimeSession, vs
             throw new Error(`Runtime metadata missing Python environment ID: ${JSON.stringify(runtimeMetadata)}`);
         }
 
-        const interpreter = interpreterService.getInterpreters().find((i) => i.id === extraData.pythonEnvironmentId);
```

Comment:
I think `yarn format-fix` run from inside `positron-python` should fix this.

Reply:
Thanks!